docs/development/style-guide.rst

```diff
@@ -0,0 +1,237 @@
+********************************************************
+Astropy Style Guide: A Writing Resource for Contributors
```

Comment:
Should we make it clear that this style guide is strictly for documentation, and not for coding?

```suggestion
Astropy Doc Style Guide: A Writing Resource for Contributors
```

Reply:
Yes, will add!